shared/numberUtils.ts

```diff
@@ -0,0 +1,27 @@
+const num =
+	'zero one two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen'.split(
+		' ',
+	);
+const tens = 'twenty thirty forty fifty sixty seventy eighty ninety'.split(' ');
+
+export const number2words = (n: number): string => {
```

Comment:
Maybe premature at this point but this feels like it could belong in [`@guardian/libs`](https://github.com/guardian/csnx/tree/main/libs/%40guardian/libs).